packages/server/src/api/todo/social/getLikedBy.ts

```diff
@@ -40,7 +43,7 @@ export default function (server: Server) {
         did: row.did,
         name: row.name,
         displayName: row.displayName || undefined,
-        createdAt: row.createdAt,
+        createdAt: dateFromDb(row.createdAt),
```

Comment:
Fixes an issue where the database communicates datetimes in the string format `YYYY-MM-DD hh:mm:ss`, but we want to communicate using ISO date strings.